packages/mobile/views/dashboard/drawers/account-actions/views/AccountActionsView.svelte

```diff
@@ -17,4 +40,25 @@
             {localize('actions.customizeAcount')}
         </Button>
     {/if}
+    {#if features?.dashboard?.accountActions?.toggleVisibility?.enabled}
+        <Button
+            outline
+            disabled={!$selectedAccount.hidden && $nonHiddenActiveAccounts.length <= 1}
+            onClick={() => ($selectedAccount.hidden ? handleShowAccountClick() : handleHideAccountClick())}
+            icon={$selectedAccount.hidden ? Icon.View : Icon.Hide}
+        >
+            {localize($selectedAccount.hidden ? 'actions.showAccount' : 'actions.hideAccount')}
+        </Button>
+    {/if}
+    {#if features?.dashboard?.accountActions?.toggleVisibility?.enabled}
```

Comment:
This should be `{#if features?.dashboard?.accountActions?.delete?.enabled}` right?

Reply:
right